### ISSUE TITLE
Element with "vertical-align: middle;" before HOT container causes white lines at the bottom of cell headers

### ISSUE THREAD

**Issue by warpech** [handsontable/handsontable#6459]
### Description

While debugging a visual glitch in an app that uses Ant Design + Handsontable, I discovered an annoying white border at the bottom of every TH in HOT.

I was able to narrow down the problem to the following steps.

At this point, I can't suggest the fix. Maybe someone knows how to reset the side effects of the external `vertical-align: middle;` within Handsontable?

### Steps to reproduce

1. Put any image before the HOT container (directly or nested in a div toolbar - it does not matter)
2. Set the CSS rule `vertical-align: middle;` on the image

### Demo

https://jsfiddle.net/fjun1tqv/

### Your environment
* Handsontable version: 7.2.2
* Browser Name and version: Chrome 78
* Operating System: Windows

### Screenshot

![image](https://user-images.githubusercontent.com/566463/68766397-125c1980-061f-11ea-935f-6f5580196317.png)

Zoom:

![image](https://user-images.githubusercontent.com/566463/68766553-5d762c80-061f-11ea-867d-b4729b33c6ea.png)

**Comment by warpech** [comment]
@krzysztofspilka says this is a problem with `background-clip: padding-box` in Handsontable style for TD and TH.

Quick workaround is to change the font size used in the element that has `vertical-align: middle;`.

**Comment by wojciechczerniak** [comment]
@warpech It's a bug in Chrome when the container is positioned with half a pixel from top or left. A better workaround would be to overwrite this CSS rule. In case something else shifts the container.

Git blame says it was introduced with fixes for IE9, so it should be safe. I will ask @swistach why it was introduced.

**Comment by warpech** [comment]
You mean somtehing like this?

```
.handsontable th,
.handsontable td {
   background-clip: initial !important; 
}
```

Thanks, this seems to do the job, too

**Comment by swistach** [comment]
As @wojciechczerniak says, `background-clip` was introduced to resolve a problem in IE9 with disappearing cells borders after selection.
Reference: #2411

**Comment by warpech** [comment]
Using the current `develop`, I can't reproduce the problem from #2411 after I remove `background-clip: padding-box;` from `walkontable.css`. It seems that this problem was solved by different means and `background-clip` can be removed from this file.

I have tested using IE11 in IE9 mode

**Comment by aninde** [comment]
Fixed by `7.4.1 v.` https://jsfiddle.net/aninde/s796anty/3/

Tested on Windows 10 / Chrome 79
![#6459-Chrome](https://user-images.githubusercontent.com/30186818/74748304-93532e80-5268-11ea-8008-ed764408a698.png)
